{"name": "Проверка ошибки авторизации пользователя при неверном логине", "status": "passed", "steps": [{"name": "Логин пользователя", "status": "passed", "parameters": [{"name": "data", "value": "{'email': 'oturner@example.org', 'password': ('bryanscott@example.com', '845#+O+7)i')}"}], "start": 1716901026977, "stop": 1716901027230}, {"name": "Проверка статуса и кода", "status": "passed", "parameters": [{"name": "code", "value": "401"}, {"name": "name", "value": "'success'"}, {"name": "text", "value": "False"}], "start": 1716901027230, "stop": 1716901027230}], "start": 1716901026976, "stop": 1716901027231, "uuid": "e26a8662-79aa-426d-a032-a06ee22e1ecb", "historyId": "b2a4903716dfa021cf0e98b359d74974", "testCaseId": "b2a4903716dfa021cf0e98b359d74974", "fullName": "tests.test_user_login.TestUserLogin#test_user_auth_with_wrong_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "Sasaraf"}, {"name": "thread", "value": "20240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}